packages/shared/src/characters/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './alice';
+export * from './bob';
+export * from './cbd-recipient';
+export * from './enrico';
+export * from './pre-recipient';
```

Comment:
What's the difference between `bob` and `pre-recipient`?

Reply:
`Bob` is a vanilla PRE character, `PreDecrypter` from `./pre-recipient` is a tdec-adapted `Bob`.

Reply:
I see, well, we should deprecate `PreDecryper` as soon as possible, but I guess that's something that can wait after this PR.

Reply:
Ok, I will deprecate the reminder of PRE-tDec